department-of-veterans-affairs/va.gov-cms
# VACMS 10046: Update Link color across the CMS to meet WCAG 2.0 AAA contrast requirement

## Description

Closes #10046.

## Testing done
Visually checked on my local machine.

## Screenshots


## QA steps

Log in and click on Content. Verify that the Overview tab text is not white and not underlined. The Overview tab itself should show the underline/bottom border. 
Verify that the edit buttons have dark gray text and background darkens on hover. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
